File: client/src/app/model/admin-staff-dto.ts
import {StaffRole} from './register-req';

export class AdminStaffDto {
   staffName:String;
   staffEmail:String;
   staffPass:String;
   staffRole:String;

  constructor(staffName: String, staffEmail: String, staffPass: String, staffRole: String) {
    this.staffName = staffName;
    this.staffEmail = staffEmail;
    this.staffPass = staffPass;
    this.staffRole = staffRole;
  }
}

File: server/src/main/java/com/upsintern/emplorium/dto/AdminStaffDto.java
package com.upsintern.emplorium.dto;

import com.upsintern.emplorium.entity.Staff;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class AdminStaffDto {
    String staffName;
    String staffEmail;
    String staffPass;
    Staff.StaffRole staffRole;
}
